recipes/r-mimic-demo/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mimic-demo
```

Let's derive the name from the library, rather than the repository:

```suggestion
  name: r-mimic.demo
```